### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [lderse/f1-3-c2p1-colmar-academy#5]
At this point each picture should be using alt text, this helps the visually impaired know what the picture is about through the use of screen readers.
